ubuntudesign/docs.ubuntu.com
# Add static API pages

So we can turn off http://docs.maas.io/2.0/api.html etc., I'm moving those api.html pages into this repo.

QA
---

``` bash
./rebuild-docs
./run
```

Visit <http://127.0.0.1:8007/maas/2.1/en/api> and <http://127.0.0.1:8007/maas/2.0/en/api>. Check they make sense.
